Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN python3.7 -m pip install --upgrade \
 
 WORKDIR /home/$USERNAME/piro/
 
-# Set ip and 
-RUN find . -name 'app.py' | xargs sed -i "s/app.run_server(debug=True)/app.run_server(host='0.0.0.0', port=8888, debug=True)/g"
+# Set ip and port of Dash app for Docker
+RUN find . -name 'app.py' | xargs sed -i "s/app.run_server(debug=True)/app.run_server(host='0.0.0.0', port=8080, debug=True)/g"
```

Comment:
Just noting that we should eventually move host/port to a config file

Reply:
Agreed, also I think we can just change app.py to have these params as defaults.